src/itemlistformaction.cpp

```diff
@@ -1508,4 +1391,127 @@ std::string ItemListFormAction::title()
 	}
 }
 
+void ItemListFormAction::handle_op_saveall() {
+	LOG(Level::INFO,
+		"ItemListFormAction: saving all items");
+
+	if (visible_items.empty()) {
+		return;
+	};
+
+	std::string directory = v->run_dirbrowser();
+
+	if (directory == "") {
+		return;
+	}
+
+	if (directory.back() != '/') {
+		directory = directory + "/";
+	}
+
+	if (visible_items.size() == 1) {
+		const std::string filename = v->get_filename_suggestion(visible_items[0].first->title());
+		const std::string fpath = directory + filename;
+
+		struct stat sbuf;
+		if (::stat(fpath.c_str(), &sbuf) != -1) {
+			std::string input_options = _("yn");
+			char c = v->confirm(strprintf::fmt(
+					_("Overwrite `%s' in `%s?' "
+					  "(y:Yes n:No)"),
+					filename, directory),
+					input_options);
+			if (!c)
```

Comment:
Curly braces here! My bad, since in the original commit I did not put them there. Sorry!